stories/Components/UIcomponents/Hero/PageHero/PageHero.stories.mdx

```diff
@@ -1,123 +1,176 @@
+          },
+          {
+            text: "عنوان الصفحة",
+          },
+        ],
+        title: "أهداف التنمية المستدامة في العمل",
+        content: "تركيزنا",
+        imgsrc: BackgroundImg,
+        imgsrc2: BackgroundImg2,
+        imgalt: BackgroundImg,
+        videosrc: BackgroundVideo,
+        cta: "اقرأ أكثر",
+      };
+      return arabicText;
+    case "burmese":
+      const burmeseText = {
+        herodata: [
+          {
+            text: "အိမ်ပြန်",
+          },
+          {
+            text: "ဒုတိယအဆင့်",
+          },
+          {
+            text: "တတိယအဆင့်",
+          },
+          {
+            text: "စာမျက်နှာခေါင်းစဉ်",
+          },
+        ],
+        title: "sdgs သည်လုပ်ဆောင်သည်",
+        content: "ကျွန်ုပ်တို့၏အာရုံ",
+        imgsrc: BackgroundImg,
+        imgsrc2: BackgroundImg2,
+        imgalt: BackgroundImg,
+        videosrc: BackgroundVideo,
+        cta: "ပိုပြီးဖတ်ပါ",
+      };
+      return burmeseText;
+    case "japanese":
+      const japaneseText = {
+        herodata: [
+          {
+            text: "家",
+          },
+          {
+            text: "セカンドレベル",
+          },
+          {
+            text: "第3レベル",
+          },
+          {
+            text: "ページタイトル",
+          },
+        ],
+        title: "動作中のsdgs",
+        content: "私たちの焦点",
+        imgsrc: BackgroundImg,
+        imgsrc2: BackgroundImg2,
+        imgalt: BackgroundImg,
+        videosrc: BackgroundVideo,
+        cta: "続きを読む",
+      };
+      return japaneseText;
     default:
-      const dummy =  {herodata: [{
-      text: 'Home'
-    },
-    {
-      text: 'Second-level'
-    },
-    {
-      text: 'Third-level'
-    },
-    {
-      text: 'Page Title'
-    }],
-    title: 'The sdgs in action',
-    content: 'Our Focus',
-    imgsrc: BackgroundImg,
-    imgsrc2: BackgroundImg2,
-    imgalt:BackgroundImg,
-    videosrc: BackgroundVideo};
-    return dummy
+      const dummy = {
+        herodata: [
+          {
+            text: "Home",
+          },
+          {
+            text: "Second-level",
+          },
+          {
+            text: "Third-level",
+          },
+          {
+            text: "Page Title",
+          },
+        ],
+        title: "The sdgs in action",
+        subtitle:
+          "Subtitle or excerpt of the story goes below and it is two lines",
+        content: "Our Focus",
+        imgsrc: BackgroundImg,
+        imgsrc2: BackgroundImg2,
+        imgalt: BackgroundImg,
+        videosrc: BackgroundVideo,
+        cta: "Read more",
+      };
+      return dummy;
   }
 };
 
-<Meta title="Components/UI components/Hero/Page"
+<Meta
+  title="Components/UI components/Hero/Page"
   argTypes={{
     variant: {
-      options: ["video", "image"],
+      options: ["video", "image", "no_background"],
       control: { type: "radio" },
-      defaultValue: 'image'
     },
-  }}/>
+    cta_enabled: {
+      control: { type: "boolean" },
+    },
+    outline_enabled: {
```

Comment:
@paboden Where you're currently using the term "outline" it should actually be "overline". The overline is the small text appearing directly over a headline. This should be renamed wherever it is used.

Reply:
Variable name has been fixed. It's been changed to "Overline" so that it also appears as "Overline" in the control UI section.